### ISSUE TITLE
Jenkins genome model test: 5.10-clinseq-wer - Build 2129 - Diffs Found

### ISSUE THREAD

**Issue by dufeiyu** [genome/genome#1596]
Project: 2-Genome-Model-Tests-2-Run-Models/TEST_SPEC=5.10-clinseq-wer
Build: https://apipe-ci.gsc.wustl.edu/job/2-Genome-Model-Tests-2-Run-Models/TEST_SPEC=5.10-clinseq-wer/2129/
Console: https://apipe-ci.gsc.wustl.edu/job/2-Genome-Model-Tests-2-Run-Models/TEST_SPEC=5.10-clinseq-wer/2129/console
********************************************************************************
DIFFERENCES FOUND:
 Comparing new object 7712d85d4956475e9649622d2b93165d to blessed object 0bc8c6ff5d4c4fc0ae645335f738a29d
  File: AML109/mutation-spectrum/b20_q10/exome/mutation_spectrum_sequence_context/AML109.prop.test.2type
  Reason: files are not the same (diff -u {/gscmnt/gc9026/test/model_data/8a175224cae14b80ab36d749216efb2b/build0bc8c6ff5d4c4fc0ae645335f738a29d,/gscmnt/gc9026/test/model_data/8a175224cae14b80ab36d749216efb2b/build7712d85d4956475e9649622d2b93165d}/AML109/mutation-spectrum/b20_q10/exome/mutation_spectrum_sequence_context/AML109.prop.test.2type)

If you want to bless this object (7712d85d4956475e9649622d2b93165d) update and commit the DB file (Model/Build/Command/DiffBlessed.pm.YAML).

**Comment by dufeiyu** [comment]
I actually checked other builds of the same model 8a175224cae14b80ab36d749216efb2b. AML109.prop.test.2type were all generated ok. I also start a brand new build f3d8c39e48ad4f9e8045e1c9ffb59134 and that file is still ok. So it seems that empty AML109.prop.test.2type in build 7712d85d4956475e9649622d2b93165d might be transient due to some glitch. I went ahead and abandon the build 7712d85d4956475e9649622d2b93165d so Jenkins will start a new build to try. Jenkins new build succeeds without diff !  I will close this issue.
